### PR TITLE
enhancement: Add support for negating conditions in check_fields

### LINKED ISSUE #2446: How to specify "not.contains" transform?
Currently there is a "contains" transform: `condition."message.contains" = "foo"`
Is it possible to specify the opposite logic: emit only messages which don't contain a certain text, something like `condition."message.not.contains" = "foo"`?

### PR DESCRIPTION
**Pull request by bruceg**
This applies to the filter and swimlanes transforms.

This negation may apply to any check_fields condition.

Signed-off-by: Bruce Guenter <bruce@timber.io>

Closes #2446

### CODE REVIEW

File: .meta/_partials/fields/_conditions_options.toml.erb [hunk 1]
```diff
@@ -12,6 +12,17 @@ is_log = "Returns true if the event is a log."
 is_metric = "Returns true if the event is a metric."
 
 
+[<%= namespace %>."`[field-name]`.not_`[condition]`"]
+type = "any"
+examples = [
+  { "message.not_contains" = "some phrase to ignore" }
```
Comment (binarylogic) [inline .meta/_partials/fields/_conditions_options.toml.erb:18]: Can we add a few more examples here?

**Review by binarylogic — APPROVED** [review]
Docs look good

**Review by lukesteensen — APPROVED** [review]
Nice! This is very neat.

**Comment by pySilver** [comment]
Great! :) This should go to docs too 👍

**Pull request merged**